docs/tutorials/02_create_your_first_app/05_using-collections.md

```diff
@@ -0,0 +1,198 @@
+---
+sidebar_position: 5
+title: Step 5 - Using collections
+---
+
+# Step 5 - Using collections
+
+In this step we will add new variable `options` to the actor that will hold a list of the choices that users can vote for in the poll.
+
+
+## Creating a "map"
+
+Each option is a string, what describes what the users are voting for. However, since we have multiple options in the poll we will need to store them in some data structure. We will use a collection called [`RBTree`](/motoko/main/base/RBTree.md) which is similar to "maps" or "dictionaries" in other languages.
```

Comment:
```suggestion
Each option is a string that describes what the user is voting for. However, since we have multiple options in the poll we will need to store them in some data structure. We will use a collection called [`RBTree`](/motoko/main/base/RBTree.md) which is similar to "maps" or "dictionaries" in other languages.
```